scripts/upgrade-facets.js

```diff
@@ -392,4 +392,9 @@ const logFacetCut = (cut, selectors) => {
   }
 };
 
+function requireUncached(module) {
```

Comment:
Could you explain why this method is needed?

Reply:
Added explanation to the code.

Reply:
thank you